frontend/src/components/DeploymentEventMessageComponent.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+  This file is part of Edgehog.
+
+  Copyright 2025 SECO Mind Srl
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  SPDX-License-Identifier: Apache-2.0
+*/
+
+import { DeploymentTargetsTable_DeploymentTargetsFragment$data } from "api/__generated__/DeploymentTargetsTable_DeploymentTargetsFragment.graphql";
+
+import { Accordion } from "react-bootstrap";
+import { useCallback, useMemo, useState } from "react";
+
+import "./DeploymentEventMessageComponent.scss";
+
+type DeploymentEventMessageComponentProps = {
+  event: NonNullable<
+    NonNullable<
+      DeploymentTargetsTable_DeploymentTargetsFragment$data[number]["deployment"]
+    >["events"]["edges"]
+  >[number]["node"];
+};
+
+const DeploymentEventMessageComponent = ({
```

Comment:
We usually avoid including the word “Component” in a component’s name.

```suggestion
const DeploymentEventMessage = ({
```